metricbeat/module/ceph/_meta/config.yml

```diff
@@ -0,0 +1,30 @@
+- module: ceph
+  metricsets: ["perf"]
+  enabled: true
+  period: 1s
+
+ceph:  
```

Comment:
Is this working? This looks like invalid yaml. All config options related to a metricset should be in the module itself.